### PR TITLE
feat: upgrade mac aws accounts 100 -> 1000

### PR DESCRIPTION
**Pull request by deimosfr**

### CODE REVIEW

File: src/aws/iam.rs [hunk 1]
```diff
@@ -114,6 +114,7 @@ impl IamService {
             .client
             .get_group()
             .group_name(iam_group.to_string())
+            .max_items(1000)
```
Comment (benjaminch) [inline src/aws/iam.rs:117]: _suggestion_: would you mind using a loop with max items = 100 and loop until there is no more users?
your solution here will break for 1001 users.

What about something like that (to be tested of course):
```
 pub async fn get_users_from_group(
        &self,
        iam_group: &IamGroup,
    ) -> Result<HashSet<AwsUser>, IamError> {
        let mut marker:Option<String> = None;
        let page_size = 100;
        let mut users: HashSet<AwsUser> = HashSet::new();

        loop {
            let response = self
                .client
                .get_group()
                .group_name(iam_group.to_string())
                .set_marker(marker)
                .max_items(page_size)
                .send()
                .await;

            match response {
                Ok(group) => {
                    let group_users = group.users();

                    if group_users.is_empty() {
                        return Err(IamError::NoUsersFoundInIamGroup {
                            group: iam_group.clone(),
                        });
                    }

                    for user in group_users {
                        let mut aws_user = AwsUser {
                            arn: Arn::new(user.arn()),
                            user_name: User::new(user.user_name()),
                            groups: HashSet::from_iter(vec![iam_group.clone()]),
                        };

                        if let Some(u) = users.get(&aws_user) {
                            aws_user.groups.extend(u.groups.clone());
                        }

                        users.insert(aws_user);
                    }

                    marker = group.marker().map(|m| m.to_string());

                    if marker.is_none() || group_users.is_empty() {
                        break;
                    }
                }
                Err(e) => {
                    return Err(IamError::CannotGetUserFromIamGroup {
                        group: iam_group.clone(),
                        raw_message: Arc::from(e.to_string()),
                    })
                }
            }
        }

        Ok(users)
    }
```

**Review by benjaminch — CHANGES_REQUESTED** [review]

File: src/aws/iam.rs (hunk 1 above)
Comment (deimosfr) [reply src/aws/iam.rs:117]: True, but I do not expect to have 1000 users without SSO to be honest :/

Comment (benjaminch) [reply src/aws/iam.rs:117]: I don't even think the `max_item` will support 1000 eventually

**Review by benjaminch — APPROVED** [review]
Vida loca :D

**Pull request merged**